draft-ietf-tsvwg-dtls-over-sctp-bis.md

```diff
@@ -905,7 +905,10 @@ terminated and the associated keying material discarded.
 
    A DTLS/SCTP Endpoint MUST NOT have more than two DTLS connections
    open at the same time. Either of the endpoints MAY initiate a new
-   DTLS connection by performing a full DTLS handshake. As either
+   DTLS connection by performing a full DTLS handshake. To support this
```

Comment:
Why do we say "full" here?

Reply:
I think the full can be removed, there are no particular reason. Any complete handshake should be sufficient here.